### PR TITLE
Move python tests from postinstall to ctest

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

File: dawn/examples/python/test_python.cmake [hunk 1]
```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.12.4)
+if(NOT DEFINED PYTHON_EXECUTABLE)
+    find_package(PythonInterp 3 REQUIRED)
```
Comment (jdahm) [inline dawn/examples/python/test_python.cmake]: This was deprecated. https://cmake.org/cmake/help/git-stage/module/FindPythonInterp.html?highlight=pythoninterp
```suggestion
    find_package(Python3 COMPONENTS Interpreter REQUIRED)
```

File: dawn/python/CMakeLists.txt [hunk 2]
```diff
@@ -48,3 +47,11 @@ else()
 
   )
 endif()
+
+if(DAWN_TESTING)
+  if(NOT DEFINED PYTHON_EXECUTABLE)
+    find_package(PythonInterp 3 REQUIRED)
```
Comment (jdahm) [inline dawn/python/CMakeLists.txt]: Also applies here.

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

**Comment by twicki** [comment]
this is deprecated since the whole python testing framework has been changed

**Pull request closed without merging**